src/datatypes/src/schema.rs

```diff
@@ -30,7 +31,7 @@ pub use crate::schema::constraint::ColumnDefaultConstraint;
 pub use crate::schema::raw::RawSchema;
 
 /// Key used to store version number of the schema in metadata.
-const VERSION_KEY: &str = "greptime:version";
+pub const VERSION_KEY: &str = "greptime:version";
```

Comment:
Looks like don't need to pub these constants now.

Reply:
Let's just keep it. It may soon be need in the follow up PR